charts/base-cluster/values.schema.json

```diff
@@ -1263,6 +1263,81 @@
       },
       "additionalProperties": false
     },
+    "weaveworks": {
+      "type": "object",
+      "description": "Values for the gitops-chart",
+      "properties": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "adminUser": {
+          "type": "object",
+          "properties": {
+            "create": {
+              "type": "boolean"
+            },
+            "username": {
+              "type": "string"
+            },
+            "password": {
+              "type": "string"
+            }
+          },
+          "if": {
+            "properties": {
+              "create": {
+                "const": true
+              }
+            },
+            "required": ["create"]
+          },
+          "then": {
+            "required": ["username","password"]
+          },
+          "additionalProperties": false
+        },
+        "ingress": {
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "type": "boolean"
+            },
+            "host": {
+              "type": "string"
+            }
+          },
+          "additionalProperties": false
+        },
+        "oidcSecret": {
+          "type": "object",
+          "properties": {
+            "create": {
+              "type": "boolean"
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "if": {
+        "properties": {
+          "enabled": {
+            "const": true
+          }
+        },
+        "required": ["enabled"]
+      },
+      "then": {
+        "anyOf": [
+          {
+            "required": ["adminUser"]
+          },
+          {
+            "required": ["oidcSecret"]
+          }
+        ]
+      },
```

Comment:
Instead of `if`, `then`, ... I'd rather just use `anyOf`

In this case this isn't even needed, as I'd set the adminUser by default to `admin` and auto-generate a password if none is set, just like for the grafana admin user

Reply:
I read a post about helm validations and json schema on stackoverflow. It was recommend to use if then instead of anyOf because it is easier to read, while the same functionality can be kept in that case

Reply:
Mh, I personally definitely don't think that it's easier to read randomly ordered json properties than plain `anyOf`

But anyways, you're still using `anyOf` *and* neither is needed here